gatsby-ssr.js

```diff
@@ -4,136 +4,60 @@ var _react = require("react");
 
 var _react2 = _interopRequireDefault(_react);
 
-function _interopRequireDefault(obj) {
-  return obj && obj.__esModule ? obj : { default: obj };
-}
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
 exports.onRenderBody = function (_ref, pluginOptions) {
   var setHeadComponents = _ref.setHeadComponents;
   var trackPage = pluginOptions.trackPage,
-    prodKey = pluginOptions.prodKey,
-    devKey = pluginOptions.devKey,
-    _pluginOptions$dataPl = pluginOptions.dataPlaneUrl,
-    dataPlaneUrl =
-      _pluginOptions$dataPl === undefined
-        ? "https://hosted.rudderlabs.com"
-        : _pluginOptions$dataPl,
-    controlPlaneUrl = pluginOptions.controlPlaneUrl,
-    delayLoad = pluginOptions.delayLoad,
-    delayLoadTime = pluginOptions.delayLoadTime,
-    manualLoad = pluginOptions.manualLoad,
-    loadType = pluginOptions.loadType,
-    useNewSDK = pluginOptions.useNewSDK,
-    useBetaSDK = pluginOptions.useBetaSDK,
-    loadOptions = pluginOptions.loadOptions,
-    sdkURL = pluginOptions.sdkURL;
+      prodKey = pluginOptions.prodKey,
+      devKey = pluginOptions.devKey,
+      _pluginOptions$dataPl = pluginOptions.dataPlaneUrl,
+      dataPlaneUrl = _pluginOptions$dataPl === undefined ? "https://hosted.rudderlabs.com" : _pluginOptions$dataPl,
+      controlPlaneUrl = pluginOptions.controlPlaneUrl,
+      delayLoad = pluginOptions.delayLoad,
+      delayLoadTime = pluginOptions.delayLoadTime,
+      manualLoad = pluginOptions.manualLoad,
+      loadType = pluginOptions.loadType,
+      useNewSDK = pluginOptions.useNewSDK,
+      sdkURL = pluginOptions.sdkURL,
+      useBetaSDK = pluginOptions.useBetaSDK,
+      _pluginOptions$loadOp = pluginOptions.loadOptions,
+      loadOptions = _pluginOptions$loadOp === undefined ? {} : _pluginOptions$loadOp;
```

Comment:
This is unnecessary. We can club them together. It applies for another thing above as well.